docs/01_menu_controls.md

```diff
@@ -1,14 +1,27 @@
 ## Menu Controls
 
-### Fast scroll
-Use the C-Up and C-Down buttons
+### Additional Control Information
 
+#### Fast scroll
+Press either the C-Up or C-Down buttons to scroll by pages, rather than by elements.
 
-### DD ROMs
+#### N64FlashcartMenu settings
+Press the START button at the browser screen to open the Settings window. From here you can edit some of the N64FlashcartMenu settings, 
+see information of either the console, the flashcart you are using or N64FlashcartMenu itself, and if your cart has Real-Time Clock (RTC) support, 
+you will also be able to change its date and time.
 
-#### Expansion Disks
-To load an expansion disk (e.g. F-Zero X) browse to the N64 ROM and load it (but not start it) and then browse to the DD expansion file and press the `R` button.
+#### Browser options
+Press the R button to open the Browser Options window. Here you can see a ROM's properties, delete it from your SD card or establish the default folder 
+where N64FlashcartMenu's browser will start in future boots.
 
-#### Disk swapping
-This feature is not currently available in the menu.
+#### Additional ROM information
+Press either the L or Z button at the ROM information screen to open an additional window that will show additional information of the currently 
+selected ROM file, such as its endianess, the regional variant, its set clock rate, and much more.
+
+### 64DD ROM
 
+#### Expansion disks
+To load an expansion disk (such as F-Zero X), first browse to the N64 ROM and load it (**but not start it!**), then browse to the 64DD expansion file and press the `R` button.
```

Comment:
Wrong now as expects the `L or Z` buttons in the latest menu.

Reply:
Oops. I forgot to correct that.